importer/importer/builder.py

```diff
@@ -389,6 +391,42 @@ def save_image(image_source_url):
         return 0
 
 
+def get_product_accountability_period(product_id: str) -> int:
+    product_endpoint = "/".join([fhir_base_url, "Group", product_id])
+    response = handle_request("GET", "", product_endpoint)
+    if response[1] == 200:
+        json_product = json.loads(response[0])
+        product_characteristics = json_product["characteristic"]
+        for character in product_characteristics:
+            if (
+                character["code"]["coding"][0]["display"]
+                == "Accountability period (in months)"
+            ):
+                accountability_period = character["valueQuantity"]["value"]
+                return accountability_period
+        logging.error(
+            "Accountability period was not found in the product characteristics : "
+            + product_id
+        )
+        return -1
+    else:
```

Comment:
Does the else [improve readability](https://www.pythonmorsels.com/unnecessary-else-statements/), or is it unnecessary?

Reply:
I wouldn't say it improves readability, removed it. 
Without the else, the "more important" part is when the response is 200. So I moved the check up as well